pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -763,36 +771,77 @@ var _ = Describe("vmexport", func() {
 
 	Context("Port-forward", func() {
 		var (
-			vme *exportv1.VirtualMachineExport
+			service *k8sv1.Service
+			pod     *k8sv1.Pod
 		)
 
 		BeforeEach(func() {
-			vme = utils.VMExportSpecPVC(vmeName, metav1.NamespaceDefault, "test-pvc", secretName)
-			vme.Status = utils.GetVMEStatus([]exportv1.VirtualMachineExportVolume{
-				{
-					Name:    "no-test-volume",
-					Formats: utils.GetExportVolumeFormat(server.URL, exportv1.KubeVirtRaw),
+			vme = &exportv1.VirtualMachineExport{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-vme",
 				},
-			}, secretName)
-			utils.HandleSecretGet(kubeClient, secretName)
-			utils.HandleVMExportGet(virtClient, vme, vmeName)
+				Spec: exportv1.VirtualMachineExportSpec{
+					TokenSecretRef: &secret.Name,
+					Source: v1.TypedLocalObjectReference{
+						APIGroup: &v1.SchemeGroupVersion.Group,
```

Comment:
API group should be used from `virtv1 "kubevirt.io/api/core/v1"`.

Reply:
Yes, right.